### PR TITLE
move v3 tokens to src/tokens

### PR DESCRIPTION
**Pull request by lukasoppermann**
## Summary

This PR copies/moves token source files to `src/tokens` to be changed without affecting v2 tokens.
It also changes the output dest for the new build flow to `tokens-v3-private/`

## What should reviewers focus on?

- does everything compile fine?
- is something missing?

## Contributor checklist:

- [ ] All new and existing CI checks pass
- [ ] Tests prove that the feature works and covers both happy and unhappy paths
- [ ] Any drop in coverage, breaking changes or regressions have been documented above
- [ ] All developer debugging and non-functional logging has been removed
- [ ] Related issues have been referenced in the PR description

## Reviewer checklist:

- [ ] Check that pull request and proposed changes adhere to our [contribution guidelines](../../CONTRIBUTING.md) and [code of conduct](../../CODE_OF_CONDUCT.md)
- [ ] Check that tests prove the feature works and covers both happy and unhappy paths
- [ ] Check that there aren't other open Pull Requests for the same update/change
- [ ] Verify the design tokens changed in this PR are expected using the diffing results in this PR

### CODE REVIEW

File: script/tempColorTokenBuild.ts
Comment (rezrah) [inline script/tempColorTokenBuild.ts:17]: I can totally see why you did this, but I think it's an optimisation at the expense of readability. I think I preferred the explicit paths as I didn't have to think at all about what it resolves to. Not blocking, but thought I'd share anyway.

File: .eslintignore
Comment (lukasoppermann) [reply .eslintignore:2]: I think the new ones we will release would be v3, yes. But this is currently to have a safe experimentation / build field. I think the CI stuff is only needed once we actually want to release it, even if experimentally, right?

File: script/tempColorTokenBuild.ts
Comment (lukasoppermann) [reply script/tempColorTokenBuild.ts:17]: Hmm, I guess it may be harder to read. I am fine reverting this.

File: .eslintignore
Comment (rezrah) [reply .eslintignore:2]: I would recommend you have it even for experimentation. I find it's most useful for maintainers like yourself and @langermank who are regularly adding and updating tokens quickly. Your call though, happy to leave it for now too.

Comment (lukasoppermann) [reply .eslintignore:2]: Oh, sure, if you can quickly add it, that would be great.

Comment (lukasoppermann) [reply .eslintignore:2]: No, I was not planning on shipping them yet.

File: scripts/tempColorTokenBuild.ts
Comment (rezrah) [inline scripts/tempColorTokenBuild.ts:202]: So this means that we're no longer supporting / issuing future updates to tokens in `tokens-v2-private/`. Wondering if we need to communicate that somewhere 🤔

Comment (lukasoppermann) [reply scripts/tempColorTokenBuild.ts]: No, because the script we use for this is unchanged. So we can still issue updates.

**Review by rezrah — APPROVED** [review]
Thanks for rolling back the change to explicit paths,  reads a lot clearer to me ☺️.

There's a merge conflict so can't confirm the deployments worked, but assuming they are.. happy to go ahead.

File: scripts/tempColorTokenBuild.ts
Comment (lukasoppermann) [reply scripts/tempColorTokenBuild.ts]: This is the build script for tokens that are currently shipped https://github.com/primer/primitives/blob/main/build.js

**Pull request merged**
